packages/vertica-nodejs/script/create-test-tables.js

```diff
@@ -68,7 +68,7 @@ con.connect((err) => {
       console.log('Created table person')
       console.log('Filling it with people')
 
-      let q = people.map((person) => `INSERT INTO person (name, age) VALUES ('${person.name}', ${person.age})`).join(';')
+      let q = people.map((person) => `INSERT INTO person (name, age) VALUES ('${person.name}', ${person.age})`).join(';') + ";COMMIT;"
```

Comment:
Is there any way we can build calls to commit into a framework?  Seems like it runs the risk of being less obvious vs more reliable and consistent.

Reply:
An alternative is just having autocommit enabled by default or set for tests when I add that property

Reply:
It would be a good feature to have. @DMickens we do not currently have this property, correct?

Reply:
I have it mostly ready to go in it's own branch, should probably just give it a test, but we've decided to curb any new features until we get all of these passing tests. I can make a note to remove any soon-to-be unnecessary commits whenever the autocommit PR makes it through.